docs/core/configuration.md

```diff
@@ -305,6 +305,15 @@ dial_timeout = "3s"
 #######################################################
 [mempool]
 
+# The type of mempool for this node to use.
+#
+#  Possible types:
+#  - "flood" : concurrent linked list mempool with flooding gossip protocol
+#  (default)
+#  - "nop"   : nop-mempool (short for no operation; the ABCI app is
+#  responsible for storing, disseminating and proposing txs)
+type = "flood"
```

Comment:
extend the comment of `create_empty_blocks`?

Reply:
https://github.com/cometbft/cometbft/pull/1643/commits/f6f3aa6bffac53d86cccc1beec391fc97de9e0a2